jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -155,3 +167,23 @@ private fun Component.setSelectionHighlighting(table: JTable, isSelected: Boolea
         background = table.background
     }
 }
+
+private class SpeedSearchHighlighter : Highlighter.HighlightPainter {
+    override fun paint(g: Graphics?, p0: Int, p1: Int, bounds: Shape?, component: JTextComponent?) {
```

Comment:
Can we fix the bad default param names?

Reply:
Sure, might as well since we have to implement it ourselves